tensorboard/webapp/widgets/line_chart_v2/line_chart_internal_utils.ts

```diff
@@ -22,44 +23,55 @@ import {isWebGl2Supported} from './lib/utils';
 /**
  * Returns extent, min and max values of each dimensions, of all data series points.
  *
+ * When ignoreYOutliers is true, it will filter out values that are approximately [1] less
+ * than 5th and greater than 95th percentile when calculating y-extent.
+ *
  * Note that it excludes auxillary data points and invisible data series.
  *
- * TODO(stephanwlee): add support for ignoreOutlier.
+ * [1]: Uses R-7 method for approximation: https://github.com/d3/d3-array#quantile
```

Comment:
FWIW, no change needed but as discussed below, the way we actually use the quantiles I think it doesn't really matter what the approximation algorithm is since we're just using the result to truncate anyway.